### ISSUE TITLE
Ladybird crash when try load russian sites

### ISSUE THREAD

**Issue by SaltedPostal** [SerenityOS/serenity#24236]
When you try to load any site that is located in the ru region, Ladybird crashes and does not load the site. backtrace pinned
(try load www.ya.ru)
[49_RequestServer_backtrace.txt](https://github.com/SerenityOS/serenity/files/15220776/49_RequestServer_backtrace.txt)

**Comment by circl-lastname** [comment]
vk.ru works, seems to be an issue specifically with Yandex's HTTP server and the HTTP implementation on our side

**Comment by circl-lastname** [comment]
That's is what I said

**Comment by trnxdev** [comment]
Maybe it's the fact that ya.ru uses Ipv6?

```
> ping ya.ru
PING ya.ru (2a02:6b8::2:242) 56 data bytes
64 bytes from ya.ru (2a02:6b8::2:242): icmp_seq=1 ttl=45 time=57.9 ms

> ping vk.ru
PING vk.ru (87.240.132.78) 56(84) bytes of data.
64 bytes from srv78-132-240-87.vk.com (87.240.132.78): icmp_seq=1 ttl=55 time=46.2 ms

> ping dzen.ru
PING dzen.ru (62.217.160.2) 56(84) bytes of data.
64 bytes from www.dzen.ru (62.217.160.2): icmp_seq=1 ttl=54 time=49.7 ms
```

**Comment by circl-lastname** [comment]
Not fixed by LadybirdBrowser/Ladybird@ab82fc8993d8da1024d9bcf84d18e8cd28fb5f22 which adds IPv6 support, meaning the issue is something else
